Documentation/ceph-cluster-crd.md

```diff
@@ -1239,3 +1240,11 @@ kubectl -n rook-ceph patch cephcluster rook-ceph --type merge -p '{"spec":{"clea
 
 Nothing will happen until the deletion of the CR is requested, so this can still be reverted.
 However, all new configuration by the operator will be blocked with this cleanup policy enabled.
+
+Rook waits for the deletion of PVs provisioned using the cephCluster before proceeding to delete the cephCluster. To force deletion of the cephCluster without waiting for the PVs to be deleted,  you can use the following command:
+
+```console
+kubectl -n rook-ceph patch cephcluster rook-ceph --type merge -p '{"spec":{"cleanupPolicy":{"uninstallMode":"yes-really-uninstall-even-if-volumes-in-use"}}}'
```

Comment:
Instead of patching it, let's just mention the setting in the CR.

Reply:
Done.